Source/WebCore/css/parser/MutableCSSSelector.h

```diff
@@ -122,7 +124,8 @@ inline bool MutableCSSSelector::needsImplicitShadowCombinatorForMatching() const
 #endif
             || pseudoElement() == CSSSelector::PseudoElement::Part
             || pseudoElement() == CSSSelector::PseudoElement::Slotted
-            || pseudoElement() == CSSSelector::PseudoElement::UserAgentPartLegacyAlias);
+            || pseudoElement() == CSSSelector::PseudoElement::UserAgentPartLegacyAlias
+            || pseudoElement() == CSSSelector::PseudoElement::WebKitUnknown);
```

Comment:
Can we move down `pseudoElement() == CSSSelector::PseudoElement::UserAgentPart` to have some alphabetic ordering?

Reply:
Does that all compile down to the same thing? I thought these might be ordered this way for perf reasons.

Reply:
We should probably add a comment if that's the case. The temptation of re-ordering is easy. 

But I doubt it makes a noticeable difference tbh

Reply:
I'm going to leave this as-is for now. There's already enough subtle changes in this patch. And there's a FIXME for this function anyway so once that's addressed this can be addressed at the same time. (Or this change simply gets undone.)

Reply:
Sounds good!